tests/counters.rs

```diff
@@ -0,0 +1,232 @@
+// These tests cannot be run in parallel. We use the "serial_test" crate to run
+// them one by one.
+
+#[cfg(test)]
+mod counters {
+    use http::StatusCode;
+    use infinispan::Infinispan;
+    use reqwest::Response;
+    use serial_test::serial;
+    use std::collections::HashSet;
+    use std::iter::FromIterator;
+
+    #[tokio::test]
+    #[serial]
+    async fn create_weak_counter() {
+        cleanup().await;
+
+        let client = infinispan_client();
+        let counter_name = "test_counter";
+
+        client
+            .run(infinispan::request::counters::create_weak(counter_name))
```

Comment:
Save your future self from this madness by `use infinispan::request::counters;` and `.run(counters::create_weak(...))`

Reply:
Done :+1: